jsonnet/telemeter/metrics.jsonnet

```diff
@@ -46,4 +46,14 @@
   // cluster alerting for the purposes of preventing upgrades from
   // failing for end users.
   '{__name__="ALERTS",alertstate="firing"}',
+  // node_cpu_utilisation is used to report the current use of CPU
+  // in the cloud.openshift.com portal.
+  '{__name__="node:node_cpu_utilisation:avg1m"}',
+  // node_memory_utilisation is used to report the current use of
+  // memory in the cloud.openshift.com portal.
+  '{__name__="node:node_memory_utilisation:ratio"}',
+  // node_filesystem_size_bytes and node_disk_utilisation are used to
+  // report the current of of storage in the cloud.openshift.com portal.
+  '{__name__="node_filesystem_size_bytes"}',
+  '{__name__="node:node_disk_utilisation:avg_irate"}',
```

Comment:
@yaacov I think that `node_disk_utilisation` is the utilisation of disk bandwidth, not the utilisation of disk space, so we will probably need a different metric.

Reply:
@jhernand , for free fs space we may use:
 '{__name__="node_filesystem_free_bytes"}'

My reason for choosing  bandwidth over space utilization is copy/paste from the grafana dashboard, I assume we will be compared to the grafana dashboard, and get BZ's on metrics that are not the same :-(

@mrsiano hi, can you help here ?

I want to keep the metrics as close as possible to the metrics shown on openshifts grafana dashboard, they use `node_disk_utilisation` , but as @jhernand mentioned above, this is the metrics for fs bandwidth and not for fs space.

Since we can not show all the metrics possible, and need to choose a very small subset of metrics to show, maybe it's better to show the fs bandwidth like the grafana dashboard ?

On the other hand, when using grafana dashboard user can see a lot more metrics, so maybe we need to choose some other metrics ?

p.s
I did not find the repository holding the grafana dashboard used by the origin console, anyone know where it is ?
It make sense to me, to keep an eye on what they are doing, so we can keep using the same metrics on our end it they update their dashboards ...